tools/pg_inspector/active_connections_to_yaml.rb

```diff
@@ -0,0 +1,98 @@
+require 'trollop'
+require 'pg'
+require 'pg_inspector/error'
+require 'pg_inspector/pg_inspector_operation'
+require 'pg_inspector/util'
+
+module PgInspector
+  class ActiveConnectionsYAML < PgInspectorOperation
+    HELP_MSG_SHORT = "Dump avtive connections to YAML file".freeze
+    def parse_options(args)
+      self.options = Trollop.options(args) do
+        opt(:pg_host, "PostgreSQL host name or address",
+            :type => :string, :short => "s", :default => "127.0.0.1")
+        opt(:user, "PostgreSQL user",
+            :type => :string, :short => "u", :default => "postgres")
+        opt(:port, "PostgreSQL server port",
+            :short => "p", :default => 5432)
+        opt(:output, "Output file",
+            :type => :string, :short => "o", :default => "active_connections.yml")
+        opt(:ignore_error, "Ignore incomplete application name column",
+            :short => "i")
+        opt(:only_miq_rows, "Get only ManageIQ Server/Worker rows",
+            :short => "m")
+      end
+    end
+
+    def run
+      Util.dump_to_yml_file(
+        filter_by_application_name(
+          pg_stat_activity_rows_to_array(
+            rows_in_pg_stat_activity(
+              connect_pg_server
+            )
+          )
+        ), "active connections", options[:output]
+      )
+    rescue Error::ApplicationNameIncompleteError => e
+      Util.error_exit(e)
+    end
+
+    private
+
+    def connect_pg_server
+      conn_options = {
+        :dbname => 'vmdb_production',
```

Comment:
Maybe better to user `postgres` here instead of `vmdb_production` because that DB should always exist. I ran it locally and it failed because I didn't have that DB.